tests/Data/AlphaNumTest.php

```diff
@@ -35,14 +35,15 @@ public static function stringValidateProvider():array{
 			['ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890 $%*+-./:', true],
 			['abc', false],
 			['ÄÖÜ', false],
-			[',', true],
+			[',', false],
 			['-', true],
 			['+', true],
 			['.', true],
 			['*', true],
 			[':', true],
 			['/', true],
 			['\\', false],
+			['0,1', false]
```

Comment:
CI is complaining about the missing comma after the array element here

Reply:
Just removed the case altogether because my issue would be detected by the previous comma-case.